src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -215,13 +215,21 @@ def __init__(
         self.model_outputs = [None] * solver_order
         self.lower_order_nums = 0
         self._step_index = None
+        self._step_index_init = None
 
     @property
     def step_index(self):
         """
         The index counter for current timestep. It will increae 1 after each scheduler step.
         """
         return self._step_index
+    
+    @property
+    def step_index_init(self):
```

Comment:
Let's add a setter method here as well (https://stackoverflow.com/questions/2627002/whats-the-pythonic-way-to-use-getters-and-setters) that can be set from all pipelines if necessary

Reply:
ok but can i ask when do we need both a setter method and this `set_begin_index` method? they are doing exactly thing, no? 
https://github.com/huggingface/diffusers/pull/5746/files#r1463161680

Reply:
Ah ok, I think we had a bit of a misunderstanding here 😅 We don't need / we shouldn't add it if it doesn't have to be used. I was under the impression that we have to use a setter method. But it seems like we don't need it after all!